docs/lib/analytics/fragments/native_common/getting-started/common.md

```diff
@@ -0,0 +1,45 @@
+The Analytics category enables you to collect analytics data for your app. The Analytics category comes with built-in support for [Amazon Pinpoint](https://aws.amazon.com/pinpoint) and [Amazon Kinesis](https://aws.amazon.com/kinesis) (Kinesis support is currently only available in the Amplify JavaScript library).
+
+## Prerequisites
+
+<inline-fragment platform="ios" src="~/lib/analytics/fragments/ios/getting-started/10_preReq.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/analytics/fragments/android/getting-started/10_preReq.md"></inline-fragment>
+
+## Set up Analytics backend
+
+Run the following command in your project's root folder. The CLI will prompt configuration options for the Analytics category such as Amazon Pinpoint resource name and analytics event settings.
+
+> The Analytics category utilizes the Authentication category behind the scenes to authorize your app to send analytics events.
+
+```bash
+amplify add analytics
+```
+
+```console
+? Provide your pinpoint resource name: `yourPinpointResourceName`
```

Comment:
Let's put these on the next line.

Reply:
Do you mean by putting an empty line between "```console" and "? Provide your *"